packages/openapi-generator/src/generator.spec.ts

```diff
@@ -1,7 +1,47 @@
-import { getSdkVersion } from './generator';
+import { resolve } from 'path';
+import { existsSync, promises } from 'fs';
+import { getSdkVersion, generate } from './generator';
 
-describe('generator', () => {
-  it('getSdkVersion returns a valid stable version', async () => {
+const { rmdir } = promises;
+
+xdescribe('generator', () => {
```

Comment:
I assume the tests are still useful as local tests.

Reply:
yes and I have a follow up ticket to bring them back. I think they are good to have beside the E2E tests we will also have soon.